.github/workflows/ci.yml

```diff
@@ -5,14 +5,20 @@ on:
     branches:
       - main
       - main_0.1
-      - vnext-prototype
-      - vnext-tokens
+      - main_0.2
```

Comment:
Should this CP be done through fluent2-tokens, THEN merged into this PR?

Reply:
ok